styleguide/source/assets/js/category-menu.js

```diff
@@ -7,14 +7,9 @@
  *
  * Licensed MIT
  */
-(function ($, Drupal) {
-  Drupal.behaviors.ama_categoryMenu = {
-    attach: function(context, settings) {
-      $('.ama_category_navigation_menu__group').smartmenus({
-        mainMenuSubOffsetX: 250,
-        mainMenuSubOffsetY: 20,
-        keepInViewport: true
-      });
-    }
-  };
-})(jQuery, Drupal);
+
```

Comment:
For some reason wrapping this function inside of a Drupal closure caused numerous javascript errors in D8